micro-integrator/values-dev.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+wso2:
+  # WSO2 subscription parameters. If you do not have an active subscription, do not provide values for the parameters.
+  # If you do not possess an active WSO2 subscription already, you can sign up for a WSO2 Free Trial Subscription at (https://wso2.com/free-trial-subscription).
+  subscription:
+    username: ""
+    password: ""
+  deployment:
+    wso2microIntegrator:
+      # Docker registry details
+      dockerRegistry: ""
+      imageName: "wso2mi"
+      imageTag: "1.2.0"
+      # Number of deployment replicas
+      replicas: 1
+      # Minimum number of seconds for which a newly created pod should be ready without any of its containers crashing, for it to be considered available.
+      minReadySeconds: 30
```

Comment:
@ThilinaManamgoda do we need this property `minReadySeconds` in WSO2 deployments? This was removed in other deployments since, [we need not wait for additional time for all the containers after the startup](https://stackoverflow.com/questions/53239081/how-does-minreadyseconds-affect-readiness-probe). WDYT?